### ISSUE TITLE
[Bug][SF][2.6.0][FlexibleColumnLayout][A11y][Expected reply Time: ASAP] Resizing/dragging action of columns missing single pointer activation for mouse and touch users

### ISSUE THREAD

**Issue by wilson-xu-sap** [SAP/ui5-webcomponents-react#6869]
### Describe the bug

The Resizing/dragging action of columns missing single pointer activation for mouse and touch users.

### Isolated Example

https://sap.github.io/ui5-webcomponents-react/v2/?path=/docs/layouts-floorplans-flexiblecolumnlayout--docs

### Expected Behaviour

A single pointer activation mechanism must be available so mouse and touch users don't have to perform the dragging operation.

### UI5 Web Components for React Version

2.6.0

### UI5 Web Components Version

2.6.0

### Browser

Chrome

### Relevant log output

```shell

```

### Declaration

- [x] I’m not disclosing any internal or sensitive information.

**Comment by MarcusNotheis** [comment]
Thanks for reporting! I'll forward this issue to our UI5 Web Components Colleagues as the affected component is developed in their repository.
